specs/merge/fork-choice.md

```diff
@@ -115,7 +115,7 @@ def validate_merge_block(block: BeaconBlock) -> None:
     if TERMINAL_BLOCK_HASH != Hash32():
         # If `TERMINAL_BLOCK_HASH` is used as an override, the activation epoch must be reached.
         assert compute_epoch_at_slot(block.slot) >= TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH
-        return block.block_hash == TERMINAL_BLOCK_HASH
+        return block.body.execution_payload.parent_hash == TERMINAL_BLOCK_HASH
```

Comment:
I think this should be `parent_hash` and not `block_hash`. Please correct me if im wrong